tpm2.0/src/getkey/main.c

```diff
@@ -8,10 +8,10 @@
  */
 
 #include "tls_util.h"
+#include "tpm2_kmyth_global.h"
 #include "tpm2_kmyth_io.h"
 #include "tpm2_kmyth_misc.h"
 #include "tpm2_kmyth_seal.h"
-#include "kmyth_log.h"
```

Comment:
I think we've gone both ways on this one, but should we leave the log include here because we're calling functions from that API?

Reply:
I added #include <kmyth_log.h> to all of the main.c files, because we use the API to configure the logging (e.g., app name, log-file path, ...). In include/tpm2_kmyth_global.h, we map the kmyth_log() macro to the log_event() function in the API, so I it needs to remain there. I don't think any of the other files use the API directly. I think they just call the kmyth_log() macro.